generate-sdks.ts

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
In other repositories we've put helper script files into a `.scripts` folder. I'd recommend either including this files contents in gulpfile.ts or moving it into a `.scripts` folder.